src/gui/windows/chest.zig

```diff
@@ -0,0 +1,73 @@
+const std = @import("std");
+
+const main = @import("main");
+const Player = main.game.Player;
+const ItemStack = main.items.ItemStack;
+const Vec2f = main.vec.Vec2f;
+const Texture = main.graphics.Texture;
+
+const gui = @import("../gui.zig");
+const GuiComponent = gui.GuiComponent;
+const GuiWindow = gui.GuiWindow;
+const Button = GuiComponent.Button;
+const HorizontalList = GuiComponent.HorizontalList;
+const VerticalList = GuiComponent.VerticalList;
+const ItemSlot = GuiComponent.ItemSlot;
+
+const hotbar = @import("hotbar.zig");
+
+pub var window = GuiWindow{
+	.relativePosition = .{
+		.{.attachedToWindow = .{.reference = &hotbar.window, .selfAttachmentPoint = .middle, .otherAttachmentPoint = .middle}},
+		.{.attachedToWindow = .{.reference = &hotbar.window, .selfAttachmentPoint = .upper, .otherAttachmentPoint = .lower}},
```

Comment:
By default, please attach it to the top of the inventory, instead of the top of the hotbar.